spec/output_formats/dot_spec.rb

```diff
@@ -1,8 +1,101 @@
 require 'spec_helper'
 require 'ronin/recon/output_formats/dot'
+require 'stringio'
 
 describe Ronin::Recon::OutputFormats::Dot do
-  it "must inherit from Ronin::Core::OutputFormats::OutputFile" do
+  subject { described_class.new(io) }
+
+  let(:io) { StringIO.new }
+  let(:values_fixutre_dir) { File.expand_path(File.join(__dir__,'..','values','fixtures')) }
+
+  it 'must inherit from Ronin::Core::OutputFormats::OutputFile' do
     expect(described_class).to be < Ronin::Core::OutputFormats::OutputFile
   end
+
+  describe '#value_type' do
+    context 'for valid value' do
```

Comment:
I think we should test all of the `Values::` classes to ensure they return the appropriate type label.